hacktober-web/README.md

```diff
@@ -1,6 +1,18 @@
 # hacktober-web
 
```

Comment:
In addition to Ryan's point, we should have some sort of "what's next" here. With the concept of this being open source, we should make sure viewers know (1.)- What we're building (2.)- How/Why we're building it and (3.)- What's next and where others can plug in.

Reply:
> In addition to Ryan's point, we should have some sort of "what's next" here. With the concept of this being open source, we should make sure viewers know (1.)- What we're building (2.)- How/Why we're building it and (3.)- What's next and where others can plug in.

I'm not even sure the answer to 1 -3 right now